README.md

```diff
@@ -45,7 +45,7 @@ a simple, free locking mechanism, and enforcing best practices around CLI usage
 
 ## Install
 
-1. Install [Terraform](https://www.terraform.io/) and make sure it is in your PATH.
+1. Install [Terraform](https://www.terraform.io/) and make sure it is in your PATH (or set `TERRAGRUNT_TFPATH`).
```

Comment:
I'd update the docs to be a bit more explicit:

Install [Terraform](https://www.terraform.io/) and make sure it is in your PATH. Alternatively, if you want to ensure you never accidentally run Terraform instead of Terragrunt, instead of putting Terraform in your PATH, specify the path to the Terraform binary via the `--terragrunt-tfpath` command line flag or the  `TERRAGRUNT_TFPATH` environment variable.